test/extended/operators/olm.go

```diff
@@ -70,4 +71,18 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-24061 - [bz 1685230] OLM operator should use imagePullPolicy: IfNotPresent
```

Comment:
It's better if you can add the author info.

Reply:
Added as suggested